mteb/abstasks/TaskMetadata.py

```diff
@@ -217,19 +237,18 @@ def _check_dataset_revision_is_specified(cls, dataset):
                 "Revision missing for the dataset %s. It is encourage to specify a dataset revision for reproducability.",
                 dataset["path"],
             )
-        return dataset
+        return True
 
-    @field_validator("eval_langs")
-    def _check_eval_langs(cls, eval_langs):
+    def eval_langs_are_valid(self, eval_langs: LANGUAGES) -> bool:
```

Comment:
Same here - the returned value is not used anywhere.
```suggestion
    def eval_langs_are_valid(self, eval_langs: LANGUAGES) -> None:
```